website/docs/pact_nirvana/step_7.md

```diff
@@ -22,36 +31,79 @@ Another reason that an extra "safe to deploy?" check is required (as well as the
 
 The `can-i-deploy` tool is a CLI that has been written to query the Matrix to ensure that you are safe to deploy. You can read a detailed description of how it works [here](/pact_broker/can_i_deploy/).
 
```

Comment:
Issue: I think it is weird to introduce the tool in step 7 when it has already been discussed extensively in step 6. I would suggest moving all the introduction part of can-i-deploy to step 6. Moving some of the links to any mention of `can-i-deploy` in step 6 would also be great!